RevenueCat.xcodeproj/xcshareddata/xcschemes/BackendIntegrationTests.xcscheme

```diff
@@ -71,15 +71,6 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES">
-      <MacroExpansion>
-         <BuildableReference
-            BuildableIdentifier = "primary"
-            BlueprintIdentifier = "2DE20B7E26409EB7004C597D"
-            BuildableName = "BackendIntegrationTestsHostApp.app"
-            BlueprintName = "BackendIntegrationTestsHostApp"
-            ReferencedContainer = "container:RevenueCat.xcodeproj">
-         </BuildableReference>
-      </MacroExpansion>
```

Comment:
Similar question here. This doesn't seem related to development docs 🤷‍♂️  Seems fine but just wanted to make sure its okay to include in this PR!

Reply:
same here, left over from the other PR, but is safe to merge